src/test/java/org/humancellatlas/ingest/submission/web/SubmissionLinkMapControllerTest.java

```diff
@@ -81,10 +93,15 @@ public void testSubmissionLinkMap() {
         assertThat(submissionLinkMap).isNotNull();
         assertThat(submissionLinkMap.processes.get("donor-specimen-process").protocols).isEqualTo(new HashSet<>(Arrays.asList("collectionProtocol")));
         assertThat(submissionLinkMap.processes.get("donor-specimen-process").inputBiomaterials).isEqualTo(new HashSet<>(Arrays.asList("donor")));
-        assertThat(submissionLinkMap.processes.get("cellSuspension-file-process").protocols).isEqualTo(new HashSet<>(Arrays.asList("sequencingProtocol")));
-        assertThat(submissionLinkMap.processes.get("cellSuspension-file-process").inputBiomaterials).isEqualTo(new HashSet<>(Arrays.asList("cellSuspension")));
+        assertThat(submissionLinkMap.processes.get("cellSuspension-sequenceFile-process").protocols).isEqualTo(new HashSet<>(Arrays.asList("sequencingProtocol")));
+        assertThat(submissionLinkMap.processes.get("cellSuspension-sequenceFile-process").inputBiomaterials).isEqualTo(new HashSet<>(Arrays.asList("cellSuspension")));
+        assertThat(submissionLinkMap.processes.get("cellSuspension-sequenceFile-process").inputFiles).isEmpty();
```

Comment:
Have you considered less duplication by using `extracting()` or `from()`?

Reply:
I believe they're not applicable here.